frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/parent-or-guardian.tsx

```diff
@@ -39,11 +39,11 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   invariant(state.dateOfBirth, 'Expected state.dateOfBirth to be defined');
   const ageCategory = getAgeCategoryFromDateString(state.dateOfBirth);
 
-  if (ageCategory !== 'children') {
+  if (ageCategory !== 'children' && ageCategory !== 'youth') {
     return redirect(getPathById('$lang+/_public+/apply+/$id+/adult/date-of-birth', params));
   }
 
-  return json({ id: state.id, csrfToken, meta });
+  return json({ id: state.id, csrfToken, meta, isYouth: ageCategory === 'youth' });
```

Comment:
Returns `ageCategory` to do the comparison in the component.

```suggestion
  return json({ id: state.id, csrfToken, meta, ageCategory });
```

Reply:
Moved the condition check inside the component (passed ageCategory in json)